lib/ui/pointer.dart

```diff
@@ -133,10 +133,21 @@ enum PointerSignalKind {
   /// A pointer-generated scale event (e.g. trackpad pinch).
   scale,
 
+  /// A stylus generated action (e.g. double tap on Apple Pencil 2)
+  stylusAction,
+
   /// An unknown pointer signal kind.
   unknown
 }
 
+  /// The preferred action for stylus action
+enum PointerPreferredAction {
```

Comment:
Should this specify stylus then?
```suggestion
enum PointerPreferredStylusAction {
```

Reply:
@hellohuanlin suggested preferredStylusAuxilliaryAction or PointerPreferredStylusAuxilliaryAction, what do you think?

Reply:
I think you have an extra "l" in "Auxiliary".  But what does "Auxiliary" mean in this context?

Reply:
I view stylus touching on screen as the "primary action" and Apple Pencil double tap as "auxiliary action". If we just call it "stylus action"  or "pointer action", it sounds like stylus touching on screen (rather finger tapping on pencil in our case)

Reply:
This would be a good question for the framework team, I'm sure they have prior art for this problem.